signin.html

```diff
@@ -19,17 +20,17 @@
           <div class="sign-inputs">
             <div class="sign-input-box">
               <label class="sign-input-label">이메일</label>
-              <input class="sign-input" />
+              <input class="sign-input" type="email" placeholder="내용 입력" />
             </div>
             <div class="sign-input-box sign-password">
-              <label class="sign-input-label">비밀번호</label>
-              <input class="sign-input" type="password" />
               <button class="eye-button" type="button">
-                <img src="./images/eye-off.png" />
+                <img id="eye-id" src="./images/eye-off.png" />
```

Comment:
r: alt 속성으로 어떤 아이콘인지 명시하는 것이 좋습니다.
```suggestion
                <img id="eye-id" src="./images/eye-off.png" alt="비밀번호 마킹 토글" />
```